### PR TITLE
EndRod,EndGateway

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by Snake1999** [comment]
Compiling passed?

**Comment by ghost** [comment]
Yes

**Comment by PikyCZ** [comment]
Can i add mobs?

File: src/main/java/cn/nukkit/block/BlockEndRod.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+ */
+public class BlockEndRod extends BlockTransparent {
+
+    public BlockEndRod() {
+        this(0);
+    }
+
+    public BlockEndRod(int meta) {
+        super(meta);
+        System.out.println(meta + "");
```
Comment (Creeperface01) [inline src/main/java/cn/nukkit/block/BlockEndRod.java:21]: what is it? :D

**Comment by ghost** [comment]
Thanks creeper :D

**Comment by vzlomka76** [comment]
@PikyCZ no, do not add mobs, this is a very heavy load on the server (Sorry, google translater)

**Comment by Creeperface01** [comment]
Mob plugin is a very heavy load

File: src/main/java/cn/nukkit/block/BlockEndRod.java (hunk 1 above)
Comment (Snake1999) [reply src/main/java/cn/nukkit/block/BlockEndRod.java:21]: I'll remove this

File: src/main/java/cn/nukkit/block/BlockEndRod.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    }
+
+    @Override
+    public double getHardness() {
+        return 2;
+    }
+
+    @Override
+    public double getResistance() {
+        return 15;
```
Comment (Snake1999) [inline src/main/java/cn/nukkit/block/BlockEndRod.java:41]: http://minecraft.gamepedia.com/End_Rod @PikyCZ 
WHO TOLD YOU THAT THE  RESISTANCE IS 15? IT IS 0 INSTEAD!!!
NEVER COPY AND PASTE CODE FROM OTHERS

**Comment by Creeperface01** [comment]
😂😂

**Pull request merged**